regions_www/m/server.R

```diff
@@ -253,64 +261,51 @@ shinyServer(function(input, output, session) {
 
     # Notify browser to update URL to reflect new region
     session$sendCustomMessage("regionchange", region$current)
+
     # Define region geography, forcing a default in cases where the geography is not available
     switch(input_purpose(),
-     "commute"= {
-      if (input$geography %in% c("msoa", "lsoa")) {
-        region_geo_change_to <- input$geography
-      } else {
-        region_geo_change_to <- "msoa"
-      }
-    },
-    "school"= {
-      region_geo_change_to <- "lsoa"
-    },
-    "alltrips"= {
-      region_geo_change_to <- "msoa"
-    })
+           "commute"= {
+             if (input$geography %in% c("msoa", "lsoa")) {
+               region_geo_change_to <- input$geography
+             } else {
+               region_geo_change_to <- "msoa"
+             }
+           },
+           "school"= { region_geo_change_to <- "lsoa" },
+           "alltrips"= { region_geo_change_to <- "msoa" }
+    )
+
     # Only trigger geography changes if required.
-    if (is.na(region$geography) || region_geo_change_to != region$geography){
-      region$geography <<- region_geo_change_to
+    if (!identical(region_geo_change_to, region$geography)) {
+      region$geography <- region_geo_change_to
     }
 
     # Set data_dir
-    region$data_dir <<- file.path(data_regional_root, input_purpose(), region$geography, region$current)
+    region$data_dir <- file.path(data_regional_root, input_purpose(), region$geography, region$current)
```

Comment:
Was always wondering why you used global assignment..

Reply:
When I added them it was `<-` https://github.com/npct/pct-shiny/commit/11e1d90ac456bbe34204eb0ffd0b042538ee3648 but it must have been changed along the way :woman_shrugging: